gutenberg.php

```diff
@@ -21,14 +21,16 @@
  * The main entry point for the Gutenberg editor. Renders the editor on the
  * wp-admin page for the plugin.
  *
+ * The gutenberg and gutenberg__editor classNames are left for backward compatibility.
```

Comment:
Do we really need it? It's going to be moved to WordPress core as far as I understand?

Reply:
Yah, all of this is going to be committed to Core.